scipy/stats/_stats_mstats_common.py

```diff
@@ -46,7 +46,8 @@ def linregress(x, y=None, alternative='two-sided'):
         intercept : float
             Intercept of the regression line.
         rvalue : float
-            Correlation coefficient.
+            The Pearson correlation coefficient. The square of ``rvalue``
+            is the coefficient of determination.
```

Comment:
```suggestion
            The Pearson correlation coefficient.
```
That part doesn't seem to be true:
https://towardsdatascience.com/r%C2%B2-or-r%C2%B2-when-to-use-what-4968eee68ed3